cmake/python-site.py

```diff
@@ -1,11 +1,11 @@
-try:
-    from setuptools.sysconfig import get_python_lib, get_config_vars
```

Comment:
This code does not work, `setuptools.sysconfig` is not a real thing. This would always result in an ImportError and skip to distutils, even if setuptools is installed.